bedrock/newsletter/templates/newsletter/opt-out-confirmation.html

```diff
@@ -50,9 +50,9 @@ <h1>{{ self.page_title() }}</h1>
               <div class="field-contents">
                 {{ form.email.errors }}
                 <label for="id_email">{{ _('Your email address:') }}</label>
-                {{ form.email }}
+                {{ field_with_attrs(form.email, placeholder=_('yourname@example.com'))|safe }}
               </div>
-              <button class="button button-hollow button-dark" type="submit">{{ _('Manage preferences') }}</button>
+              <button class="button button-hollow button-dark" type="submit">{{ _('Manage Preferences') }}</button>
```

Comment:
Do we need to alert L10n here?

Reply:
This page isn't localized, so shouldn't impact l10n